app/src/main/java/org/mozilla/rocket/privately/PrivateModeActivity.kt

```diff
@@ -269,9 +269,10 @@ class PrivateModeActivity : BaseActivity(),
     }
 
     private fun checkShortcutPromotion(continuation: () -> Unit) {
+        val isHomeScreen = screenNavigator.topFragment?.tag == ScreenNavigator.HOME_FRAGMENT_TAG
```

Comment:
add a comment to explain why it's needed. since the boolean parameter is always confusing.